in_toto/runlib.py

```diff
@@ -641,8 +641,20 @@ def in_toto_record_start(step_name, material_list, signing_key=None,
     base_path (optional): A path relative to which artifacts are recorded.
         Default is the current working directory.
 
+    command (optional): A list consisting of a command and arguments executed
```

Comment:
I can see a slight case for this to go into `stop`, while we're pinging @lukpueh. But I don't think it's a big deal and we generally want to lean towards users knowing what command(s) they're running at `start` time. Definitely not something to block this PR!

Reply:
Why not move all new arguments to `stop`? This would make the change quite a bit smaller.